app/views/images/_thumbnail.html.erb

```diff
@@ -0,0 +1,42 @@
+<div class="dz-preview dz-image-preview" data-image-id="<%= image.present? ? image.id : '' %>">
+  <% if image.present? %>
```

Comment:
this file is the html template used by dropzone for new thumbnails. I made it a partial so that we could show existing images in the dropzone as thumbnails matching dropzone's style, and I used erb instead of slim cause it was a lot of copy-pasted html.